app.py

```diff
@@ -71,11 +72,18 @@ def assets(path):
 AZURE_COSMOSDB_DATABASE = os.environ.get("AZURE_COSMOSDB_DATABASE")
 AZURE_COSMOSDB_ACCOUNT = os.environ.get("AZURE_COSMOSDB_ACCOUNT")
 AZURE_COSMOSDB_CONVERSATIONS_CONTAINER = os.environ.get("AZURE_COSMOSDB_CONVERSATIONS_CONTAINER")
-AZURE_COSMOSDB_ACCOUNT_KEY = os.environ.get("AZURE_COSMOSDB_ACCOUNT_KEY")
+AZURE_COSMOSDB_ACCOUNT_KEY = os.environ.get("AZURE_COSMOSDB_ACCOUNT_KEY") + "=="
```

Comment:
why is the trailing "==" needed ?

Reply:
Yeah, curious why they did that. Cosmos keys do end with `==` but I don't know why someone would have copied only the characters up to that.

Reply:
I'll take that out - I meant to take that out earlier. It was the only way I could get the key to work.

Reply:
Fixed - thanks for catching.

Reply:
It would be good to understand (if it's even possible) why they added it. Would it be better to not remove it but instead strip "==" from the string before appending it? That way it would (or should) work for their scenario as well as ours.